### DISCUSSION TITLE
Welcome to witnet-rust Discussions!

### DISCUSSION THREAD

**Discussion by aesedepece** [witnet/witnet-rust#1961] [General]
## 👋 Welcome!
  We’re trying GitHub's new _Discussions_ feature as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by skyfullofdiamondz** [comment]
Hey y’all, John-Michael is my name, and I usually use GitHub as a resource for locating worthwhile projects for investment and in searching for useful apps. I just found blockchain and Cryptocurrencies in August of 2020, however, I have spent literally every waking moment since seeking to explore and grasp these concepts, while reveling in Decentralization, and foolishly losing everything while attempting to learn futures on the fly. This project appeals to me as at surface level it seems to have further developed the ability to expand in  real-time capacity the complexity of incentivized honesty (trustless, I believe it’s referred to as in this space). Here primarily to learn, hopefully eventually earn too, once able to acquire what’s needed to set-up a WIT node. Would love to meet more people in the community, introduce yourself and or say hi if you’re inclined.

**Reply by aesedepece** [reply]
Welcome @skyfullofdiamondz!

Here are some useful community links:

- Join the community [Telegram group](https://t.me/witnetio) and [Discord server](https://discord.gg/QKEa5gU) 💬
- Follow [@witnet_io](https://twitter.com/witnet_io) on Twitter 🐦
- Read the [Witnet whitepaper](https://arxiv.org/pdf/1711.09756v1.pdf) 📃
- Read the [Witnet documentation](https://docs.witnet.io)❓
- Subscribe to the [Witnet Foundation blog](https://medium.com/witnet) 📬

**Reply by skyfullofdiamondz** [reply]
Hey! @aesedepece, I really appreciate your response and that you are gracious enough to make time for a unskilled person like myself to provide some guidance. I haden't checked back because most within this community are so busy with their noses in the air to usually notice. Additionally,the best projects are usually unnoticed until they after they have re-shaped the how the world works accordingly. So I didn't expect much traffic. You are literally designing the abilities for this whole market, Blockchain/Cryptocurrency to move forward and more freely access parts of the world that will serve most optimal for it's integration. How does that feel man...? I am a big fan. Your e-citizen insight has led me down a rabbit-trail this past few weeks. I look forward to any updates of what your up to now or next.
